src/agent/src/rpc.rs

```diff
@@ -1803,8 +1815,35 @@ mod tests {
     use super::*;
     use crate::protocols::agent_ttrpc::AgentService as _;
     use oci::{Hook, Hooks};
+    use tempfile::tempdir;
     use ttrpc::{r#async::TtrpcContext, MessageHeader};
 
+    // Parameters:
+    //
+    // 1: expected Result
+    // 2: actual Result
+    // 3: string used to identify the test on error
+    macro_rules! assert_result {
```

Comment:
Ideally we'd create a local crate to avoid the duplication (redefining this macro), but a quick look shows that might be harder than it appears, so I think this is ok for now, particularly given this is in the test code.